dhis-2/dhis-services/dhis-service-validation/src/main/java/org/hisp/dhis/dataanalysis/jdbc/JdbcDataAnalysisStore.java

```diff
@@ -287,4 +302,102 @@ public JdbcDataAnalysisStore( StatementBuilder statementBuilder,
 
         return jdbcTemplate.query( sql, new DeflatedDataValueNameMinMaxRowMapper( null, null ) );
     }
+
+    /**
+     * HQL query used to select {@link FollowupValue}s.
+     */
+    private static final String FOLLOWUP_VALUE_HQL = "select new org.hisp.dhis.dataanalysis.FollowupValue(" +
```

Comment:
Maybe an opinion but I would prefer more than one column per line, makes it easier and more natural to read.

Reply:
I'll group them by table